src/order/filter/pi.lean

```diff
@@ -0,0 +1,191 @@
+/-
+Copyright (c) 2021 Yury G. Kudryashov. All rights reserved.
```

Comment:
Not sure who you want to credit here.

Reply:
I wrote the part about `filter.pi` (proofs are new, not copied from the code about `nhds`). According to `git blame`, Alex wrote `filter.Coprod`.

Reply:
Yeah, I meant between you and Alex. You will have written two thirds of the file, so I guess you get priority.

Reply:
I wrote the header before moving `Coprod` here. I don't care about its contents.